tox.ini

```diff
@@ -82,5 +82,5 @@ deps=
   teamcity-messages
 changedir=packages/bootstrap/extra
 commands=
-  python setup.py install
+  pip install .
```

Comment:
here we should be running inside of a python 3.4 virtual environment, so _should_ be safe. That said, be a nice after this lands misc. cleanup PR (Trying to land this quick to un-break)